exporter/src/main/java/co/rsk/core/db/bootstrap/FileExporter.java

```diff
@@ -124,7 +126,7 @@ private static MapDBBlocksIndex getBlockIndex(String databaseDir) {
     }
 
     private static byte[] encodeState(TrieStoreImpl originUnitrieStore, Block block) {
-        Trie node = originUnitrieStore.retrieve(block.getStateRoot());
+        Trie node = originUnitrieStore.retrieve(block.getStateRoot()).get();
```

Comment:
Maybe also out of scope, but the `get` method could raise an exception if the element is not present. Usually in master code we explicitly checked this situation.

Reply:
Would we like to do anything in particular if the element is not present?

Reply:
Maybe an exception with a detailed message (ie having the state root hash description), not needed yet in this tool